src/js/components/Select/StyledSelect.js

```diff
@@ -8,9 +9,10 @@ import {
   selectedStyle,
   controlBorderStyle,
   sizeStyle,
+  styledComponentsConfig,
 } from '../../utils';
 
-export const StyledContainer = styled.div`
+export const StyledContainer = styled(Box)`
```

Comment:
@jcfilben Can you remind me why in some cases we changed to "Box" vs others we kept as div?

would be good to document this decision for future reference.

Reply:
There were some places that were using `styled.div` and `as={Box}` so I think we decided to just simplify to `styled(Box)`